### PR TITLE
Signature renames

### PR DESCRIPTION
**Pull request by dblevins**
There are two commits in this PR:

1. remove any false jakarta packages
2. add any missed jakarta renames

It seems some of these signature files are no longer needed and should be removed.  I don't have enough knowledge to pick through and know which are still needed and which are not, so opting just to fix them all (needed or not).

Very willing to remove the no-longer-needed signature files if someone can give me a list, but more than likely it'd be easier for someone else to just remove them than tell someone else which files to remove.

### CODE REVIEW

File: install/jakartaee/bin/sig-test-pkg-list_se11.txt [hunk 1]
```diff
@@ -164,19 +164,19 @@ jakarta.xml.bind.annotation.adapters
...
-jakarta.xml.registry
-jakarta.xml.registry.infomodel
-jakarta.xml.rpc
-jakarta.xml.rpc.encoding
-jakarta.xml.rpc.handler
-jakarta.xml.rpc.handler.soap
-jakarta.xml.rpc.holders
-jakarta.xml.rpc.server
-jakarta.xml.rpc.soap
+javax.xml.registry
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test-pkg-list_se11.txt:167]: javax.xml.registry + javax.xml.rpc are pruned from EE 9, so probably should be removed.

File: install/jakartaee/bin/sig-test_se8.map [hunk 2]
```diff
@@ -143,15 +143,15 @@ jakarta.resource=2.0_se8
 
 # JAXR 1.0 
 # (This is JSR-093)
-#jakarta.xml.registry=1.0_se8
+#javax.xml.registry=1.0_se8
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test_se8.map:146]: JAXR is pruned from EE 9 so probably should remove javax.xml.registry

File: install/jakartaee/bin/sig-test_se8.map [hunk 3]
```diff
@@ -143,15 +143,15 @@ jakarta.resource=2.0_se8
...
 
 # Java EE Management 1.1 
 # (This is JSR-077)
-#jakarta.management.j2ee=1.1_se8
+#javax.management.j2ee=1.1_se8
 
 # Java EE Deployment 1.2 
 # (This is JSR-088)
-#jakarta.enterprise.deploy=1.2_se8
+#javax.enterprise.deploy=1.2_se8
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test_se8.map:154]: Deployment is also pruned so probably should remove javax.enterprise.deploy

File: install/jakartaee/bin/sig-test-pkg-list_se8.txt [hunk 4]
```diff
@@ -164,19 +164,19 @@ jakarta.xml.bind.annotation.adapters
...
-jakarta.xml.registry
-jakarta.xml.registry.infomodel
-jakarta.xml.rpc
-jakarta.xml.rpc.encoding
-jakarta.xml.rpc.handler
-jakarta.xml.rpc.handler.soap
-jakarta.xml.rpc.holders
-jakarta.xml.rpc.server
-jakarta.xml.rpc.soap
+javax.xml.registry
```
Comment (scottmarlow) [inline install/jakartaee/bin/sig-test-pkg-list_se8.txt:167]: javax.xml.registry + javax.xml.rpc should be removed as they are pruned in EE 9.

**Comment by scottmarlow** [comment]
I added a few comments, probably worth waiting for Alwin to review also (he knows more about the signature test internals than I do).

File: install/jakartaee/bin/sig-test-pkg-list_se11.txt (hunk 1 above)
Comment (dblevins) [reply install/jakartaee/bin/sig-test-pkg-list_se11.txt]: Should be removed now

File: install/jakartaee/bin/sig-test_se8.map (hunk 2 above)
Comment (dblevins) [reply install/jakartaee/bin/sig-test_se8.map]: Should be removed now

File: install/jakartaee/bin/sig-test_se8.map (hunk 3 above)
Comment (dblevins) [reply install/jakartaee/bin/sig-test_se8.map]: Should be removed now

File: install/jakartaee/bin/sig-test-pkg-list_se8.txt (hunk 4 above)
Comment (dblevins) [reply install/jakartaee/bin/sig-test-pkg-list_se8.txt]: Should be removed now

**Comment by dblevins** [comment]
Thanks for the quick review @scottmarlow !  @alwin-joseph looking forward to your thoughts.

File: install/jakartaee/bin/sig-test-pkg-list_se8.txt [hunk 5]
```diff
@@ -34,7 +34,7 @@ java.util
 java.util.concurrent
 jakarta.activation
 jakarta.annotation
-jakarta.annotation.processing
+javax.annotation.processing
```
Comment (alwin-joseph) [inline install/jakartaee/bin/sig-test-pkg-list_se8.txt:37]: +1

File: install/jakartaee/bin/sig-test-pkg-list_se8.txt [hunk 6]
```diff
@@ -137,7 +135,7 @@ jakarta.servlet.jsp.jstl.sql
 jakarta.servlet.jsp.jstl.tlv
 jakarta.servlet.jsp.tagext
 jakarta.transaction
-jakarta.transaction.xa
+javax.transaction.xa
```
Comment (alwin-joseph) [inline install/jakartaee/bin/sig-test-pkg-list_se8.txt:138]: +1

File: install/jakartaee/bin/sig-test_se8.map [hunk 7]
```diff
@@ -141,18 +138,6 @@ jakarta.mail=2.0_se8
 # (This is JSR-322)
 jakarta.resource=2.0_se8
 
-# JAXR 1.0
```
Comment (alwin-joseph) [inline install/jakartaee/bin/sig-test_se8.map:144]: They were commented out, but good to remove.

File: install/jpa/bin/sig-test.map [hunk 8]
```diff
@@ -23,17 +23,17 @@
 # Note: Recording the signatures of a package includes all
 #       child packages.  The signature test tool looks for
 #       the best signature file to use when playing back
-#       signatures.  Meaning if we have a javax.servlet
```
Comment (alwin-joseph) [inline install/jpa/bin/sig-test.map:26]: ok to remove this file

File: install/jms/bin/sig-test.map [hunk 9]
```diff
@@ -34,10 +34,10 @@
 # <package>.sig_<technology_version><JavaSE_version_these_work_with>
 # Given this format, the following example breaks down as follows:
 #    jakarta.jms.sig_2.0_se8:
-#      <package> = javax.jms
```
Comment (alwin-joseph) [inline install/jms/bin/sig-test.map:37]: ok to remove this file

File: install/jsp/bin/sig-test.map [hunk 10]
```diff
@@ -26,4 +26,4 @@
 ###
 
 # JSP 2.X 
-javax.servlet.jsp=2.3_se8
```
Comment (alwin-joseph) [inline install/jsp/bin/sig-test.map:29]: ok to remove this file

File: install/jta/bin/sig-test.map [hunk 11]
```diff
@@ -23,4 +23,4 @@
 ###############################################################
 
 # JTA 1.3 packages
-javax.transaction=1.3_se8
```
Comment (alwin-joseph) [inline install/jta/bin/sig-test.map:26]: can remove this file.

File: install/saaj/bin/sig-test.map [hunk 12]
```diff
@@ -13,4 +13,4 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 #
-javax.xml.soap=1.3_se6
```
Comment (alwin-joseph) [inline install/saaj/bin/sig-test.map:16]: can remove this file

File: install/securityapi/bin/sig-test.map [hunk 13]
```diff
@@ -31,17 +31,17 @@
...
+#       signatures.  Meaning if we have a jakarta.servlet
+#       signature file and a jakarta.servlet.jsp signature file,
+#       the signature test tool will use the jakarta.servlet.jsp
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
 #       The signatures are in both files (since the API Check
 #       tool records child package signatures and there does
 #       not seem to be a way to turn this feature off) but the
-#       javax.servlet.jsp signature file can vary independent
```
Comment (alwin-joseph) [inline install/securityapi/bin/sig-test.map:43]: can remove this file as install/securityapi/bin/sig-test_se8.map & install/securityapi/bin/sig-test_se11.map will be used.

File: install/servlet/bin/sig-test.map [hunk 14]
```diff
@@ -17,4 +17,4 @@
 ###############################################################
 # The signature test mapping file for the Servlet TCK.
 ###############################################################
-javax.servlet=4.0_se8
```
Comment (alwin-joseph) [inline install/servlet/bin/sig-test.map:20]: can remove this file

File: install/jaxrs/bin/sig-test.map [hunk 15]
```diff
@@ -17,4 +17,4 @@
 ###############################################################
 # The signature test mapping file for the JAX-RS TCK.
 ###############################################################
-javax.ws.rs=1.1
```
Comment (alwin-joseph) [inline install/jaxrs/bin/sig-test.map:20]: ok to remove this file

File: install/jaspic/bin/sig-test.map [hunk 16]
```diff
@@ -31,17 +31,17 @@
...
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
 #       The signatures are in both files (since the API Check
 #       tool records child package signatures and there does
 #       not seem to be a way to turn this feature off) but the
-#       javax.servlet.jsp signature file can vary independent
-#       of the javax.servlet signature file.
+#       jakarta.servlet.jsp signature file can vary independent
+#       of the jakarta.servlet signature file.
```
Comment (alwin-joseph) [inline install/jaspic/bin/sig-test.map:44]: can remove this file.

File: install/jakartaee/bin/sig-test-pkg-list_se11.txt [hunk 17]
```diff
@@ -164,19 +164,10 @@ jakarta.xml.bind.annotation.adapters
...
-jakarta.xml.rpc.handler
-jakarta.xml.rpc.handler.soap
-jakarta.xml.rpc.holders
-jakarta.xml.rpc.server
-jakarta.xml.rpc.soap
 jakarta.xml.soap
-jakarta.xml.transform
-jakarta.xml.transform.dom
-jakarta.xml.transform.sax
+javax.xml.transform
```
Comment (alwin-joseph) [inline install/jakartaee/bin/sig-test-pkg-list_se11.txt:168]: +1

File: install/jacc/bin/sig-test.map [hunk 18]
```diff
@@ -24,17 +24,17 @@
...
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
 #       The signatures are in both files (since the API Check
 #       tool records child package signatures and there does
 #       not seem to be a way to turn this feature off) but the
-#       javax.servlet.jsp signature file can vary independent
-#       of the javax.servlet signature file.
+#       jakarta.servlet.jsp signature file can vary independent
+#       of the jakarta.servlet signature file.
```
Comment (alwin-joseph) [inline install/jacc/bin/sig-test.map:37]: can remove this file.

File: install/connector/bin/sig-test.map [hunk 19]
```diff
@@ -30,17 +30,17 @@
...
+#       signature file to verify the jakarta.servlet.jsp package
+#       signatures even though the jakarta.servlet signature
+#       file contains the jakarta.servlet.jsp package signatures.
 #       The signatures are in both files (since the API Check
 #       tool records child package signatures and there does
 #       not seem to be a way to turn this feature off) but the
-#       javax.servlet.jsp signature file can vary independent
-#       of the javax.servlet signature file.
+#       jakarta.servlet.jsp signature file can vary independent
+#       of the jakarta.servlet signature file.
```
Comment (alwin-joseph) [inline install/connector/bin/sig-test.map:43]: can remove this file

File: install/concurrency/bin/sig-test.map [hunk 20]
```diff
@@ -31,12 +31,12 @@
 # with the following format:
 # <package>.sig_<technology_version>_<JavaSE_version_these_work_with>
 # Given this format, the following example breaks down as follows:
-#    javax.enterprise.concurrent.sig_1.0_se8:
-#      <package> = javax.enterprise.concurrent
+#    jakarta.enterprise.concurrent.sig_1.0_se8:
+#      <package> = jakarta.enterprise.concurrent
```
Comment (alwin-joseph) [inline install/concurrency/bin/sig-test.map:35]: can remove this file.

**Review by alwin-joseph — COMMENTED** [review]
For most of the technologies the mapping file used is "sig-test_se8.map"(for run with jdk8) and the old "sig-test.map" still has references to old technology version (except for install/jstl/bin/sig-test.map which is still the mapping file for jstl). As per the sigtest runs it will look for sig-test_se8.map file first and if it is not present will fall back to use sig-test.map. 

So IMO we could remove those old "sig-test.map" as I commented (except install/jstl/bin/sig-test.map).

File: install/jpa/bin/sig-test.map (hunk 8 above)
Comment (dblevins) [reply install/jpa/bin/sig-test.map:26]: Deleted

File: install/jms/bin/sig-test.map (hunk 9 above)
Comment (dblevins) [reply install/jms/bin/sig-test.map:37]: Deleted

File: install/jaspic/bin/sig-test.map (hunk 16 above)
Comment (dblevins) [reply install/jaspic/bin/sig-test.map]: Deleted

File: install/jsp/bin/sig-test.map (hunk 10 above)
Comment (dblevins) [reply install/jsp/bin/sig-test.map:29]: Deleted

File: install/jta/bin/sig-test.map (hunk 11 above)
Comment (dblevins) [reply install/jta/bin/sig-test.map:26]: Deleted

File: install/saaj/bin/sig-test.map (hunk 12 above)
Comment (dblevins) [reply install/saaj/bin/sig-test.map:16]: Deleted

File: install/securityapi/bin/sig-test.map (hunk 13 above)
Comment (dblevins) [reply install/securityapi/bin/sig-test.map:43]: Deleted

File: install/servlet/bin/sig-test.map (hunk 14 above)
Comment (dblevins) [reply install/servlet/bin/sig-test.map:20]: Deleted

File: install/jaxrs/bin/sig-test.map (hunk 15 above)
Comment (dblevins) [reply install/jaxrs/bin/sig-test.map:20]: Deleted

File: install/jacc/bin/sig-test.map (hunk 18 above)
Comment (dblevins) [reply install/jacc/bin/sig-test.map]: Deleted

File: install/connector/bin/sig-test.map (hunk 19 above)
Comment (dblevins) [reply install/connector/bin/sig-test.map]: Deleted

File: install/concurrency/bin/sig-test.map (hunk 20 above)
Comment (dblevins) [reply install/concurrency/bin/sig-test.map]: Deleted

**Comment by dblevins** [comment]
@alwin-joseph Thank you so much for the explanation on how the sig-test.map files relate to their equivalent "se8" and "se11" versions.

I've gone ahead and done the deletions of any `sig-test.map` files and ensured there was both an se8 and se11 file for each (i.e. added one for jstl).  I took an educated guess this also applied to `sig-test-pkg-list.txt` files and did the same alignment there.  If I went too far I made sure that was a separate commit that is easy to back out.  Let me know.

I started to wonder while making the change if we're causing a problem for people who want to test on Java 14, 15, etc.  Will the signature tests fail for them without a fallback?  If yes, should we universally rename either se11 or se8 back to the fallback name? (perhaps the 11 set as they are most in line with 14, 15, etc)

**Comment by alwin-joseph** [comment]
> @alwin-joseph Thank you so much for the explanation on how the sig-test.map files relate to their equivalent "se8" and "se11" versions.
> 
> I've gone ahead and done the deletions of any `sig-test.map` files and ensured there was both an se8 and se11 file for each (i.e. added one for jstl). I took an educated guess this also applied to `sig-test-pkg-list.txt` files and did the same alignment there. If I went too far I made sure that was a separate commit that is easy to back out. Let me know.
> 
Thanks lets keep it.

> I started to wonder while making the change if we're causing a problem for people who want to test on Java 14, 15, etc. Will the signature tests fail for them without a fallback? If yes, should we universally rename either se11 or se8 back to the fallback name? (perhaps the 11 set as they are most in line with 14, 15, etc)

Test run on >JDK11 would ideally also require us to regenerate new signatures (as also described here https://github.com/eclipse-ee4j/jakartaee-tck/issues/156 ). As you have guessed the mapping file helps to identify the signature file to be used for the test. It should be ok to keep the se11 map file (which will point to the se11 sig file) as a fall back for jdk14/15 but we cannot guarantee the test passing for those.

**Review by alwin-joseph — APPROVED** [review]

**Comment by dblevins** [comment]
What would we need to do to enable someone to fall back on those 11 files when running on JDK 14/15?  Understood we cannot guarantee tests will pass for jdk14/15 or even 11.

**Comment by alwin-joseph** [comment]
> What would we need to do to enable someone to fall back on those 11 files when running on JDK 14/15? Understood we cannot guarantee tests will pass for jdk14/15 or even 11.

We can copy sig-test-pkg-list_se11.txt & sig-test_se11.map to sig-test-pkg-list.txt & sig-test.map respectively to enable this.


https://github.com/eclipse-ee4j/jakartaee-tck/blob/beaf767aac9b3edf330638c27b4d1927b08450fe/src/com/sun/ts/tests/signaturetest/SignatureTestDriver.java#L116
https://github.com/eclipse-ee4j/jakartaee-tck/blob/beaf767aac9b3edf330638c27b4d1927b08450fe/src/com/sun/ts/tests/signaturetest/SignatureTestDriver.java#L63

**Comment by dblevins** [comment]
@alwin-joseph Perfect.  I'll submit a second PR that re-adds default files based on the 11 files.

**Pull request merged**